src/madt.rs

```diff
@@ -328,7 +328,7 @@ struct GicInterruptTranslationServiceEntry {
 
 pub(crate) fn parse_madt<H>(
     acpi: &mut Acpi,
-    handler: &mut H,
+    _handler: &mut H,
```

Comment:
Is there a reason for this change?

Reply:
This change is done by `cargo fix --edition` command. It may be because the field is not used.

Reply:
I see